src/components/material.js

```diff
@@ -14,6 +15,16 @@ var warn = debug('components:material:warn');
 var MATERIAL_TYPE_BASIC = 'MeshBasicMaterial';
 var MATERIAL_TYPE_STANDARD = 'MeshStandardMaterial';
 
+var stereoscopicOffsets = {
+  OU: { l: 0, r: 0.5 },
```

Comment:
This is texture offset value. 
Such as repeat, offset are uv calucation value. (ranging from 0 to 1)
(or coding style?)